AdvancedRocketry/blocks/MicrowaveReciever.php

```diff
@@ -25,9 +25,10 @@
     $description1 = '<p>Generates power beamed down from Solar satellites placed in orbit.</p>
     <p>To connect a satellite to the Microwave Reciever the satellite chip must be placed in one of the item input hatches built into the structure.  This allows the satellite to track where to send the energy.</p>
     <p>The amount of power generated per tick is directly related to the power generated by linked satellites.</p>
-    <p>These satellites must be in orbit around the same planet that the Microwave reciever is built on otherwise the satellites will be unable to transmit power.</p>
+    <p>These satellites must be in orbit around the same planet that the Microwave Reciever is built on otherwise the satellites will be unable to transmit power.</p>
     <p>Even if the Microwave Reciever is turned off or not connected to a satellite, the satellites in orbit will continue to generate power which can be stored on a satellite\'s internal buffer.</p>
     <p>The reciever itself can be toggled on and off using the GUI in the Microwave Reciever control block</p>
+    <p>Because of the efficiency of the satellites placed in geosynchronous orbit and their constant view of the sun, the Microwave Reciever can generate a large amount of power, appropriate for the amount of glowstone, redstone, and gold invested into the multiblock and it\'s satellites. It is reccomended to have as many input hatches on the multiblock as can fit, so the maximum amount of satellites can be connected to one multiblock reciever.</p>
```

Comment:
```suggestion
    <p>Because of the efficiency of the satellites placed in geosynchronous orbit and their constant view of the sun, the Microwave Receiver can generate a large amount of power, appropriate for the amount of glowstone, redstone, and gold invested into the multiblock and it\'s satellites. It is recommended to have as many input hatches on the multiblock as can fit, so the maximum amount of satellites can be connected to one multiblock reciever.</p>
```